services/.rubocop.yml

```diff
@@ -554,7 +554,7 @@ Style/DocumentDynamicEvalDefinition:
   Enabled: true
 
 Style/EachWithObject:
-  Enabled: true
+  Enabled: false
```

Comment:
I find that `reduce` is much more intuitive than `each_with_object`. Both aliases point to a function that has reducing behavior.